ebuild-maintenance/git/text.xml

```diff
@@ -202,11 +202,15 @@ way. Additionally, consistency in message format allows for easier
 parsing by external tools. The first line of the commit message should
 contain a brief summary of the changes, followed by an empty new
 line. From the third line onward should be a detailed, multiline
-explanation of the changes introduced by the commit. At the very end,
-auxiliary information such as the bugs related to the commit,
-contributors, and reviewers should be listed using RFC822/git style
-tags. The length of the lines in the message should be 70-75
-characters at maximum.
+explanation of the changes introduced by the commit. Following that,
```

Comment:
Overall LGTM. The order of things don't naturally appear that way though. See the following example commit:

```
commit b48f5e83b285aa086a844e1e9bd1035d49df122a
Author:     Thomas Deutschmann <whissi@whissi.de>
AuthorDate: Fri Sep 6 16:57:52 2019 +0200
Commit:     Göktürk Yüksek <gokturk@gentoo.org>
CommitDate: Wed Oct 16 14:47:48 2019 -0400

    general-concepts/dependencies: Fix ranged dependencies example
    
    Closes: https://github.com/gentoo/devmanual.gentoo.org/pull/106
    Acked-by: Michael Orlitzky <mjo@gentoo.org>
    Signed-off-by: Thomas Deutschmann <whissi@whissi.de>
    Signed-off-by: Göktürk Yüksek <gokturk@gentoo.org>
```

Because of the way git parses the `--signoff` option, it'll append the `Signed-off-by` after `Closes:`. I'd probably merge the signoff sentences with the sentence that starts with "At the very end,", and also perhaps talk about repoman doing this automatically and passing `--signoff` to git if not using repoman.

If you don't want to edit any further, that's fine. I can make those changes post merge.

Reply:
Ok, made it a little different. Put most of the sign-off action at the section dealing with doing commits. Feels more correct that way since that is the point you are actually committing to the sign-off.
At that point, the information about the sign-off in the commit message functions more as a verification for the reader. And to finish it up, added the line in the example.